### PR TITLE
feat(circleci): snapshot testing for documentation website

### PR DESCRIPTION
**Pull request by emmenko**
Source: https://docs.percy.io/docs/command-line-client#section-generating-snapshots

Let's see how good it works, to snapshot a statically built website.

### CODE REVIEW

**Comment by emmenko** [comment]
Nice!

image

image

**Comment by emmenko** [comment]
What do you guys think?

**Comment by emmenko** [comment]
Btw, to give a bit more context about the reasoning: I want to be able to catch regressions like this one: #933

**Comment by emmenko** [comment]
Now percy runs 2 builds

image

Not sure how the status check exactly reflects that...maybe it only keeps the "last" one? @montezume do you know about this maybe?

**Comment by emmenko** [comment]
We could also maybe look into "merging" the static website and the UI components pages...

**Comment by emmenko** [comment]
Ah, I think we need to set up different projects: https://docs.percy.io/docs/multiple-projects-per-repo

**Comment by montezume** [comment]
@emmenko this would be my assumption of what has to happen. I'm not sure Percy supports multiple Percy runs for one monorepo...

**Comment by emmenko** [comment]
I created a separate Percy project to target the website snapshots. Let's see how it goes...

**Comment by emmenko** [comment]
Nice, seems to work! 🎉

File: .circleci/config.yml [hunk 1]
```diff
@@ -97,10 +97,10 @@ commands:
           command: PERCY_TOKEN="$PERCY_TOKEN_UI_COMPONENTS" yarn vrt
       - run:
           name: Building Documentation Website
-          command: yarn --cwd website build
+          command: BUILD_MODE=percy yarn --cwd website build
```
Comment (tdeekens) [inline .circleci/config.yml]: Nit: `BUILD_TARGET` maybe be more apt?

**Pull request merged**